glide.yaml

```diff
@@ -56,6 +56,8 @@ import:
 - package: gopkg.in/cheggaaa/pb.v1
 - package: github.com/asaskevich/govalidator/
   version: 5
+- package: github.com/DATA-DOG/godog
+  version: c3f5ce27930fff8efffaaf0acab7f24d5deecdbc
```

Comment:
@gbraad I can see godog actually have release tag https://github.com/DATA-DOG/godog/releases can we use that instead of hash?